src/main/sensors/battery.c

```diff
@@ -270,42 +269,34 @@ void updateCurrentMeter(int32_t lastUpdateAt, rxConfig_t *rxConfig, uint16_t dea
 
             updateCurrentDrawn(lastUpdateAt);
 
-            updateConsumptionWarning();
```

Comment:
I don't know, probably got merged this way in one of the many rebases I had to do while this PR is still open.